source/codegen/templates/service_helpers.mako

```diff
@@ -756,6 +756,7 @@ ${set_response_values(normal_outputs, init_method)}\
   config = data['config']
   input_parameters = [p for p in parameters if common_helpers.is_input_parameter(p)]
   resource_handle_types = service_helpers.get_resource_handle_types(config)
+  error_parameter = common_helpers.get_parameter_for_error_generation(parameters)
```

Comment:
```suggestion
  input_parameters = [p for p in parameters if common_helpers.is_input_parameter(p)]
  output_parameters = [p for p in parameters if common_helpers.is_output_parameter(p)]
  resource_handle_types = service_helpers.get_resource_handle_types(config)
```

Reply:
FYI, you could filter using `"return_on_error_key" in p` here too, or filter using `%for` / `%if`. It doesn't matter much either way.